goyek.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Can this live in the `build` directory?

Reply:
It would be less user-friendly as `goyek.sh` and `goyek.ps1` are convenience scripts to run the build pipelines.
Then the user would have to run `./build/goyek.sh`  or `./internal/build/goyek.sh` to run the default task. I think for similar reason the `Makefile` is in the root so that the user does not have run e.g. `make -f ./build/Makefile`